whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java

```diff
@@ -580,6 +583,22 @@ public String getDescription(final ObjectType objectType) {
         }
     }
 
+    class GeofeedSyntax implements AttributeSyntax {
+
+        @Override
+        public boolean matches(ObjectType objectType, String value) {
+            final String[] validProtocol = new String[] {"https"};
```

Comment:
Move up to a static field like ```VALID_PROTOCOLS``` (if using an array, name as plural)?

Reply:
Moved the whole validator class out of the method.